client/components/SimilarListings.jsx

```diff
@@ -0,0 +1,47 @@
+import React from 'react';
+import $ from 'jquery';
+
+import Listing from './Listing.jsx';
+import sampleData from './sampledata';
+
+class SimilarListings extends React.Component {
+  constructor(props) {
+    super(props);
+
+    this.state = {
+      listings: sampleData,
+    };
+  }
+
+  componentDidMount() {
+    const url = window.location.pathname;
+
+    if (url !== '/') {
+      $.ajax({
```

Comment:
maybe use [fetch](https://developer.mozilla.org/en-US/docs/Web/API/Fetch_API/Using_Fetch) instead so you don't need to load jQuery just for 1 function.

Reply:
Thanks, that is a great suggestion! I will definitely refactor to that if I have time, but for now will use the jquery since it's already installed and I want to make sure I meet the component and unit tests deadline.